homework/homework24/page_locators/add_contact_page.py

```diff
@@ -0,0 +1,21 @@
+"""Module for locator storage"""
+
+
+class AddContactLocators:
+    """The class contains locators to add a contact"""
+    # buttons
+    add_contact_btn = '[id="add-contact"]'
```

Comment:
Define locators in a correct way, first part:
```suggestion
    add_contact_btn = (By.CSS_SELECTOR, "add-contact")
```

Reply:
fixed, update files with locators